models/Element/Tag/Dao.php

```diff
@@ -292,4 +292,38 @@ public function getElementsForTag(
 
         return $elements;
     }
+
+    /**
+     * @param string $tagPath separated by "/"
+     *
+     * @return null|Tag
+     */
+    public function getByPath($tagPath) {
+        $parentTagId = 0;
+
+        $tag = null;
+        foreach (explode('/', $tagPath) as $tagItem) {
```

Comment:
It should work with same value return by `getNamePath()` which is with leading slash e.g. `"/key1/key2/key2"`

Reply:
Fixed.